Assets/Scripts/GameAI/AIAgentsUtil.cs

```diff
@@ -17,11 +17,11 @@ public void Init(AIAgentManager aiAgentManager)
         public float GetClosestAgentDistance()
         {
             float closestDistance = float.MaxValue;
-
+            /*
```

Comment:
Why is this commented out?

Reply:
This was part of my commenting out all the things from the merge that weren't working. I thought I got all of them. My bad.